openrtb_ext/convert_up.go

```diff
@@ -37,24 +38,24 @@ func ConvertUpTo26(r *RequestWrapper) error {
 // are no access errors.
 func convertUpEnsureExt(r *RequestWrapper) error {
 	if _, err := r.GetRequestExt(); err != nil {
-		return fmt.Errorf("req.ext is invalid: %v", err)
+		return &errortypes.BadInput{Message: fmt.Sprintf("req.ext is invalid: %v", err)}
```

Comment:
No issue with making these more specific. Just want to check in that the new type is being used in the code / that this change (here, and the rest of this PR) matters.

Reply:
In the ORTB error response, the NBR value indicates the reason why Prebid could not return a response. The purpose of using specific error types is to provide a precise NBR value that clearly identifies the cause of the failure.